### PR TITLE
Changes GBP payouts

### PR DESCRIPTION
**Pull request by Swiftfeather**
## About The Pull Request

Payouts that I believe better reflect us being a downstream. \

## Why it's good

Encourages more fixes, balances, and general qol updates, while not punishing for features as heavily.

## Changelog

This is a github change. No cl necessary.

### CODE REVIEW

**Comment by Swiftfeather** [comment]
> You currently have a negative Fix/Feature pull request delta of -19. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

Pain.

**Comment by YakumoChen** [comment]
honestly why are we even using GBP here? It's more or less ignored anyway.

**Comment by NullDagaf** [comment]
Ah yes, let's go and """balance""" the fuck out of everything tor the sake of these SWEET internet points.

File: .github/gbp.toml [hunk 1]
```diff
@@ -3,17 +3,17 @@ reset_label = "GBP: Reset"
 
 [points]
 "Atomic" = 2
-"Balance/Rebalance" = -8
+"Balance/Rebalance" = 2
```
Comment (Gandalf2k15) [inline .github/gbp.toml]:
```suggestion
"Balance/Rebalance" = -4
```

**Pull request merged**